language/vm/vm-runtime/vm-cache-map/src/cache_map.rs

```diff
@@ -19,30 +23,31 @@ pub type CacheRefMap<'a, K, V> = CacheMap<'a, K, V, &'a V>;
 /// TODO: should the map own the arena?
 pub struct CacheMap<'a, K, V, W> {
     alloc: &'a Arena<V>,
-    map: CHashMap<K, W>,
+    map: Mutex<HashMap<K, W>>,
```

Comment:
Could you add a TODO comment here explaining that this may eventually use a concurrent hashmap pending benchmarking?

Reply:
Done